lib/mutant/mutator/node/regexp/greedy_zero_or_more.rb

```diff
@@ -0,0 +1,21 @@
+module Mutant
+  class Mutator
+    class Node
+      module Regexp
+        # Mutator for greedy zero-or-more quantifier, `*`
+        class GreedyZeroOrMore < Node
+          handle(:regexp_greedy_zero_or_more)
+
+          # Emit mutations
+          #
+          # Replace `/a*/` with `/a+/`
+          #
+          # @return [undefined]
+          def dispatch
+            emit(s(:regexp_greedy_one_or_more, *children))
```

Comment:
Ah, you're right. Sorry for the oversight.

> we should have a generic forcing function / spec to recurse into children by default.

Interesting, I wonder what that would look like...